frontend/src/citizen-frontend/children/sections/service-need-and-daily-service-time/AttendanceSummaryTable.tsx

```diff
@@ -0,0 +1,148 @@
+          <IconButton
+            icon={faChevronLeft}
+            onClick={() =>
+              setAttendanceSummaryDate(attendanceSummaryDate.subMonths(1))
+            }
+            aria-label={t.calendar.previousMonth}
+          />
+          <div>{attendanceSummaryDate.formatExotic('MM / yyyy')}</div>
+          <IconButton
+            icon={faChevronRight}
+            onClick={() =>
+              setAttendanceSummaryDate(attendanceSummaryDate.addMonths(1))
+            }
+            aria-label={t.calendar.nextMonth}
+          />
+        </FixedSpaceRow>
+      </ListGrid>
+      {combine(serviceNeedsResponse, attendanceSummaryResponse).mapAll({
+        failure: () => <ErrorSegment title={t.common.errors.genericGetError} />,
+        loading: () => <Spinner />,
+        success: ([serviceNeeds, attendanceSummary]) => (
+          <AttendanceSummary
+            serviceNeeds={serviceNeeds.filter(
+              (sn) =>
+                attendanceSummaryRange.overlaps(
+                  new FiniteDateRange(sn.startDate, sn.endDate)
+                ) && sn.contractDaysPerMonth !== null
+            )}
+            attendanceSummary={attendanceSummary}
+          />
+        )
+      })}
+    </>
+  )
+})
+
+interface AttendanceSummaryProps {
+  serviceNeeds: ServiceNeedSummary[]
+  attendanceSummary: AttendanceSummary
+}
+
+const AttendanceSummary = ({
+  serviceNeeds,
+  attendanceSummary: { plannedDays, realizedDays }
+}: AttendanceSummaryProps) => {
+  const t = useTranslation()
+  return (
+    <>
+      {serviceNeeds.length > 0 ? (
+        serviceNeeds.map(({ startDate, contractDaysPerMonth }) => {
+          const plannedWarning =
+            contractDaysPerMonth !== null && plannedDays > contractDaysPerMonth
+          const realizedWarning =
+            contractDaysPerMonth !== null && realizedDays > contractDaysPerMonth
+          return (
+            <React.Fragment key={startDate.formatIso()}>
+              <ListGrid>
+                <Label>{t.children.attendanceSummary.planned}</Label>
+                <span>
+                  <Days warning={plannedWarning}>{plannedDays}</Days> /{' '}
+                  {contractDaysPerMonth} {t.common.datetime.dayShort}
+                </span>
+                <Label>{t.children.attendanceSummary.realized}</Label>
+                <span>
+                  <Days warning={realizedWarning}>{realizedDays}</Days> /{' '}
+                  {contractDaysPerMonth} {t.common.datetime.dayShort}
+                </span>
+              </ListGrid>
+              {(plannedWarning || realizedWarning) && (
+                <AlertBox message={t.children.attendanceSummary.warning} />
+              )}
+            </React.Fragment>
+          )
+        })
+      ) : (
+        <div>{t.children.attendanceSummary.empty}</div>
+      )}
+    </>
+  )
+}
+
+const Days = styled.span<{ warning: boolean }>`
+  ${(p) =>
+    p.warning &&
+    `
+    color: ${colors.status.warning}
+  `}
+`
```

Comment:
not sure if this works correctly or should it rather be `p.warning ? 'color: ...' : ''` to avoid `false` appearing in css

Reply:
It should work correctly per https://styled-components.com/docs/advanced#tagged-template-literals:

> during flattening, styled-components ignores interpolations that evaluate to undefined, null, false, or an empty string (""), which means you're free to use [short-circuit evaluation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#Short-circuit_evaluation) to conditionally add CSS rules.